src/Chainweb/Rosetta/Util.hs

```diff
@@ -220,7 +220,7 @@ rowDataToAccountLog :: AccountRow -> Maybe AccountRow -> AccountLog
 rowDataToAccountLog (currKey, currBal, currGuard) prev = do
   case prev of
     Nothing ->
-      -- ^ First time seeing account
+      -- First time seeing account
```

Comment:
Unrelated, but breaks my build. New version of Haddock are more picky about invalid markup.